packages/salesforcedx-vscode-core/package.json

```diff
@@ -116,15 +116,15 @@
       "explorer/context": [
         {
           "command": "sfdx.force.apex.class.create",
-          "when": "explorerResourceIsFolder && resourceFilename != aura && resourceFilename != lwc && sfdx:project_opened"
+          "when": "explorerResourceIsFolder && resourceFilename == classes && sfdx:project_opened"
```

Comment:
This is good but looks like it breaks a test so need to get that updated as well.

Reply:
Looks good to me too. The test failure doesn't look related to these changes, I was seeing that fail locally for me on another branch as well...

Reply:
@brpowell Ah that, I was stumped on that one. I couldn't figure out why this change broke some random localization test.